source/captiveportal/captive-portal-configuration.rst

```diff
@@ -0,0 +1,171 @@
+Configuring a Captive Portal Zone
+=================================
+
+When creating a captive portal zone, multiple settings can be edited,
+depending on your needs : 
+
+
+Avaliable timeouts
+------------------
+
+pfSense's captive portal has 3 integrated timeout options : 
+
+- **"Idle timeout"** causes users to be automatically disconnected after an amount of time of network inactivity.
+  The "last activity time of each user" is retrived using the last byte sent by each device. 
+  (meaning last byte "uploaded" from the user perspective)
+- **"Hard timeout"** causes users to be automatically disconnected after a certain amount of time, regardless of their activity.
```

Comment:
`Users will be automatically disconnected after a defined time period, whether or not they are active.`